packages/taco/test/taco-sign.test.ts

```diff
@@ -0,0 +1,214 @@
+import { PorterClient, SigningCoordinatorAgent } from '@nucypher/shared';
+import { fakePorterUri } from '@nucypher/test-utils';
+import { ethers } from 'ethers';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+
+import { signUserOp } from '../src/sign';
+
+describe('TACo Signing', () => {
+  let signUserOpMock: ReturnType<typeof vi.fn>;
+  let mockProvider: ethers.providers.Provider;
+
+  beforeEach(() => {
+    signUserOpMock = vi.fn();
+    mockProvider = {} as ethers.providers.Provider;
+    
+    vi.spyOn(PorterClient.prototype, 'signUserOp').mockImplementation(signUserOpMock);
+    vi.spyOn(SigningCoordinatorAgent, 'getParticipants').mockResolvedValue([
+      { operator: '0x1234', provider: '0x5678', signature: '0x90ab' },
+      { operator: '0xabcd', provider: '0xefgh', signature: '0xijkl' }
+    ]);
+    vi.spyOn(SigningCoordinatorAgent, 'getThreshold').mockResolvedValue(2);
+  });
+
+  describe('signUserOp', () => {
+    const userOp = {
+      sender: '0x123',
+      nonce: '0x1',
+      factory: '0x0',
+      factoryData: '0x',
+      callData: '0xabc',
+      callGasLimit: '0x20000',
+      verificationGasLimit: '0x15000',
+      preVerificationGas: '0x1000',
+      maxFeePerGas: '0xabc',
+      maxPriorityFeePerGas: '0x123',
+      paymaster: '0x0',
+      paymasterVerificationGasLimit: '0x0',
+      paymasterPostOpGasLimit: '0x0',
+      paymasterData: '0x',
+      signature: '0x',
+    };
+
+    it('should sign a user operation with zerodev account', async () => {
```

Comment:
zero dev and kernel aren't real types. Any reason you need 2 tests instead of just 1. How about 1 test that uses "0.8.0".